Source/Common/JSONPool.h

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+
+#include <FEXCore/fextl/memory.h>
+#include <FEXCore/fextl/list.h>
+
+#include <tiny-json.h>
+
+namespace FEX::JSON {
+struct JsonAllocator {
+  jsonPool_t PoolObject;
+  fextl::list<json_t> json_objects;
+
+  JsonAllocator();
+};
+static_assert(offsetof(JsonAllocator, PoolObject) == 0, "This needs to be at offset zero");
+
+json_t* PoolInit(jsonPool_t* Pool);
+json_t* PoolAlloc(jsonPool_t* Pool);
```

Comment:
These functions don't need to be declared in the header file.

Reply:
Fixed, thanks